src/components/fields/chips/chips.tsx

```diff
@@ -61,6 +62,16 @@ export const Chips = (props: IGenericFieldProps<IChipsSchema>) => {
 		// eslint-disable-next-line react-hooks/exhaustive-deps
 	}, [validation]);
 
+	useDeepCompareEffect(() => {
+		const updatedValues = [];
+		options.forEach((option) => {
+			if (value?.includes(option.value)) {
+				updatedValues.push(option.value);
+			}
+		});
```

Comment:
I feel that it would be more intuitive to act on the values e.g. `value.filter(() => options.find())`

Reply:
ok have updated